### PR TITLE
new ideas branch (my bad LMAO)

### PR DESCRIPTION
**Pull request by rowelsabahat**

### CODE REVIEW

File: src/views/IdeasView.vue
Comment (daniel-panhead) [inline src/views/IdeasView.vue:1]: buttons are very sexy, but could you add a transition for the SVG in the idea card to match up with the rest of the card? Right now it instantly changes color on hover which is different from the rest of the card

**Review by daniel-panhead — CHANGES_REQUESTED** [review]

File: src/views/IdeasView.vue
Comment (rowelsabahat) [reply src/views/IdeasView.vue:1]: fixed it (maybe)

Comment (daniel-panhead) [reply src/views/IdeasView.vue:1]: yahoo

**Review by daniel-panhead — APPROVED** [review]
sexy smexy

**Comment by daniel-panhead** [comment]
Fixed a random merge error of literal whitespace

**Pull request merged**